haystack/nodes/prompt/prompt_node.py

```diff
@@ -765,9 +765,9 @@ def __init__(
         :param use_auth_token: The authentication token to use for the model.
         :param use_gpu: Whether to use GPU or not.
         :param devices: The devices to use for the model.
-        :param top_k: Number of independently generated text to return per prompt.
-        :param stop_words: Stops text generation if any one of the stop words is generated.
-        :param model_kwargs: Additional keyword arguments passed when loading the model specified by `model_name_or_path`.
+        :param top_k: The number of independently generated texts to return per prompt. For example, if you set top_k=3, the model's going to generate three answers to the query. 
```

Comment:
How about this slightly altered point? 

For example, if you set top_k=3, the model's going to generate three responses per prompt, where it used to generate only one by default.